CommonLibSF/src/RE/T/TESObjectREFR.cpp

```diff
@@ -9,6 +9,18 @@ namespace RE
 		return func(this);
 	}
 
+	bool GetItemCount_Int(TESObjectREFR** TargetREF, TESForm* MyForm, std::uint64_t pad, float* result);
+
+	float TESObjectREFR::GetItemCount(TESForm* MyForm)
```

Comment:
Follow the casing standard for arguments etc. in this example it should be `a_form`

Also fix func above and in .h file. As well as local variable in func, can use camelCase or snake_case for local

Reply:
makes sense.
Done.